src/models/lineChart.js

```diff
@@ -54,6 +54,13 @@ nv.models.lineChart = function() {
             y = yAxis.tickFormat()(lines.y()(e.point, e.pointIndex)),
             content = tooltip(e.series.key, x, y, e, chart);
 
+        if (e.sibs) {
+            for (var i = 0; i < e.sibs.length; i++) {
```

Comment:
This is a good place to use "for in" instead of having to use the length property, this also makes it able to handle objects if you wanted to switch to that later.